keras/src/backend/jax/distributed_backend.py

```diff
@@ -0,0 +1,95 @@
+import jax
+import jax.lax as lax
+
+
+def get_device_info():
```

Comment:
Arguably, this use case is already covered by `keras.distribution.list_devices()`.

Also, I don't see this being used by subsequent PRs. Did I miss it?